### PR TITLE
See also links rendering without title text in guides

### PR DESCRIPTION
**Pull request by tahiya-mongo**
Jira: https://jira.mongodb.org/browse/DOP-579

Staging: https://docs-mongodborg-staging.corp.mongodb.com/master/guides/tahiya.chowdhury/DOP-579/

### CODE REVIEW

**Comment by tahiya-mongo** [comment]
see https://github.com/mongodb/snooty/pull/168

File: src/templates/guide.js [hunk 1]
```diff
@@ -19,7 +19,11 @@ export default class Guide extends Component {
     super(propsFromServer);
 
     const {
-      pageContext: { __refDocMapping },
+      pageContext: {
+        __refDocMapping,
+        // eslint-disable-next-line no-unused-vars
+        metadata: { slugToTitle: slugTitleMapping },
```
Comment (sophstad) [inline src/templates/guide.js]: [nit] Probably no need to [rename](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assigning_to_new_variable_names) this variable.

File: src/templates/guide.js [hunk 2]
```diff
@@ -19,7 +19,11 @@ export default class Guide extends Component {
     super(propsFromServer);
 
     const {
-      pageContext: { __refDocMapping },
+      pageContext: {
+        __refDocMapping,
+        // eslint-disable-next-line no-unused-vars
```
Comment (sophstad) [inline src/templates/guide.js]: I think you'll be able to remove this lint suppression once you change line 143.

File: src/templates/guide.js [hunk 3]
```diff
@@ -136,7 +140,7 @@ export default class Guide extends Component {
           refDocMapping={getNestedValue(['__refDocMapping'], pageContext) || {}}
           addTabset={this.addGuidesTabset}
           pillstrips={pillstrips}
-          slugTitleMapping={pageContext.slugTitleMapping}
+          slugTitleMapping={pageContext.metadata.slugToTitle}
```
[the remark this answers is not in the document]
Comment (tahiya-mongo) [reply src/templates/guide.js]: For some reason, this change is giving me a "'slugToTitle' is not defined”
error

**Pull request merged**